src/server.tsx

```diff
@@ -3,6 +3,8 @@ import { Request, Response } from "express";
 import React from "react";
 import { renderToNodeStream } from "react-dom/server";
 import { Transform } from "stream";
+// tslint:disable-next-line
+const { Readable } = require('stream');
```

Comment:
`import { Transform, Readable } from "stream";` doesn't work?

Reply:
Could have done this but the issue would have been on line 203 with `scriptStream = Readable.from(script);` because the method from is not in the typescript declaration even if the method exists.

Thought that it better to do this and disable tslint for next line instead of disable on line 203.

What do you think ?

Reply:
LGTM m. We should report this missing definition upstream!

Reply:
@dunglas found the problem. I missed the needed package. 
I added the package @types/node that contain the definition. 

On my local, build and tests are ok

:)